mailbox/postgres/src/main/java/org/apache/james/mailbox/postgres/quota/JPAPerUserMaxQuotaDAO.java

```diff
@@ -1,238 +0,0 @@
-/****************************************************************
```

Comment:
Please kill MaxDomainMessageCount MaxDomainStorage MaxGlobalMessageCount MaxGlobalStorage MaxUserMessageCount and MaxUserStorage as well